src/components/Modal/Modal.test.jsx

```diff
@@ -0,0 +1,247 @@
+import { mount } from 'enzyme';
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+
+import { Button } from '../Button';
+import Modal from './Modal';
+import ModalEvent from './ModalEvent';
+
+describe('<Modal />', () => {
+
+    let root;
+
+    beforeEach(() => {
+        root = document.createElement('div');
+        root.setAttribute('id', 'root');
+        document.body.appendChild(root);
+    });
+
+    afterEach(() => {
+        document.body.removeChild(root);
+        document.body.innerHTML = '';
+        root = null;
+    });
+
+    const defaultTitle = 'modal title';
+    const okButton = <Button key="ok" id="ok-button">OK</Button>;
+    const defaultChildren = (
+        <div>
+            <Modal.Body>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</Modal.Body>
+            <Modal.Controls>{okButton}</Modal.Controls>
+        </div>   
+    );
+
+    const component = (overrides={}) => {
```

Comment:
I missed this before, but this helper function makes these tests significantly different from all of the others and while it does make this test have a little bit less code it makes them a lot less readable.  I'll approve the change as-is but this might be something we consider refactoring before release.